hw-model/src/lib.rs

```diff
@@ -582,8 +582,17 @@ pub trait HwModel {
 
     /// Toggle reset pins and wait for ready_for_fuses
     fn warm_reset(&mut self) {
-        // sw-emulator lacks support: https://github.com/chipsalliance/caliptra-sw/issues/540
```

Comment:
This function should be overridden in the ModelEmulated impl rather than here, as ModelEmulator has direct access to the emulator internals. That will let you avoid this hack writing though generic-output-wires.